src/AddIns/Uno.UI.Lottie/LottieVisualSource.wasm.cs

```diff
@@ -70,7 +70,7 @@ public partial class LottieVisualSource
 			{
 				var documentPath = string.IsNullOrEmpty(UNO_BOOTSTRAP_APP_BASE)
 					? UriSource?.PathAndQuery
-					: UNO_BOOTSTRAP_APP_BASE + "/" + UriSource?.PathAndQuery;
+					: UNO_BOOTSTRAP_APP_BASE + UriSource?.PathAndQuery;
```

Comment:
Thanks for that big refactoring of the whole framework.

Reply:
![image](https://user-images.githubusercontent.com/33458222/89587290-5e836580-d80f-11ea-81a9-2823f9ec963b.png)